test/test.js

```diff
@@ -188,21 +198,10 @@ describe('rollup-plugin-babel', function() {
 			});
 	});
 
-	it('warns about deprecated usage with external-helper plugin', () => {
-		/* eslint-disable no-console */
-		const messages = [];
-		const originalWarn = console.warn;
-		console.warn = msg => {
-			messages.push(msg);
-		};
-		return bundle('samples/external-helpers-deprecated/main.js').then(() => {
-			console.warn = originalWarn;
-
-			assert.deepEqual(messages, [
-				'Using "external-helpers" plugin with rollup-plugin-babel is deprecated, as it now automatically deduplicates your Babel helpers.',
-			]);
+	it('allows using in combination with @babel/plugin-external-helpers', () => {
```

Comment:
Oh neat! Can you walk me through how these changes result in us supporting `@babel/plugin-external-helpers` again? I'm far less familiar with this use-case.

Reply:
`@babel/plugin-external-helpers` assumes that helpers are available on `babelHelpers` object and that it's reachable in the final scope (so most commonly - that it is a global), it's probably relict of the past and not used by many, but at least we can bring "support" back for it